reth-extension/src/pool/pool.rs

```diff
@@ -186,21 +186,43 @@ where
         let (_, tx) = self.validate(origin, transaction).await;
         self.pool.add_transaction_and_subscribe(origin, tx)
     }
+
     /*
      * Scalaris
-     * Todo: Inject abci client to send transaction out
+     * Inject abci client to send transaction out
      */
     async fn add_transaction(
         &self,
         origin: TransactionOrigin,
         transaction: Self::Transaction,
     ) -> PoolResult<TxHash> {
-        let (_, tx) = self.validate(origin, transaction).await;
-        let mut results = self.pool.add_transactions(origin, std::iter::once(tx));
-        results
-            .pop()
-            .expect("result length is the same as the input")
+        use crate::abci::abci_ext::AbciMessage;
```

Comment:
use crate:: should be placed in the header area. I have searched all the code base and I don't see any "use crate" or "use " in the middle of a function - then we should make it consistent in all code base